libs/datasets/sources/fips_population.py

```diff
@@ -40,6 +40,7 @@ class Fields(object):
 
     def __init__(self, path):
         data = pd.read_csv(path, dtype={"fips": str})
+        data['fips'] = data.fips.str.zfill(5)
```

Comment:
Can you move this into the `standardize_data` function? would prefer to have all of that logic there

Reply:
Also, i'm curious which fips didn't have padded zeros? 
when I run, it shows they're all length 5 (or na for states):
```
pop = FIPSPopulation.local().population().data.fips
pop.groupby(pop.apply(lambda x: x if pd.isna(x) else len(x))).size()
```

Reply:
this whole needs to be re-done

Reply:
I would rather have a check that explicitly breaks if the fips isn't len 5 I think. but just my 0.02